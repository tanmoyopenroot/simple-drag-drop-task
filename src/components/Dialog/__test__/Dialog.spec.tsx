import * as React from 'react';
import { mount } from 'enzyme';

import {
  Dialog,
  IDialogProps,
} from '../Dialog';

describe('Testing <Button />', () => {
  const dialogRef = React.createRef<HTMLDialogElement>();

  const wrapper = mount<IDialogProps, {}>(
    (
      <Dialog ref={dialogRef}>
        Some Content
      </Dialog>
    ),
  );

  it('should be able to render with props', () => {
    expect(wrapper.length).toEqual(1);
    expect(wrapper.children().length).toBe(1);
    expect(wrapper.text()).toEqual('Some Content');
    expect(wrapper.find('[forwardedRef]').text()).toEqual('Some Content');

    expect(wrapper.html()).toMatchSnapshot();
  });
});
