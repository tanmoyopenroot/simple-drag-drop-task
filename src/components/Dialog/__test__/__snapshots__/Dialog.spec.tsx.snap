// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Button /> should be able to render with props 1`] = `ReactWrapper {}`;
